csv-quiz-timer/quiz/quiz.go

```diff
@@ -0,0 +1,40 @@
+package quiz
+
+import (
+	"encoding/csv"
+	"fmt"
+	"os"
+	"strings"
+)
+
+type Problem struct {
+	Question, Answer string
+}
+
+// Handles runtime errors
+func handleError(e error, message string) {
+	if e != nil {
+		fmt.Println(message)
+		os.Exit(1)
```

Comment:
I would rather see you propagate errors from this helper function and call out the `os.Exit` only on main file. This would make it easier to debug.